_manual-config/main.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
nit: I'd add some package-level documentation here about the purpose of this CLI tool